Doc/library/linecache.rst

```diff
@@ -60,6 +67,12 @@ The :mod:`linecache` module defines the following functions:
 
    .. versionadded:: 3.5
 
+
+.. function:: updatecache(filename, module_globals=None)
+
+   Update a cache entry and return its list of lines.
+   If something's wrong, print a message, discard the cache entry, and return an empty list.
```

Comment:
It would be great if you can detail what is the meaning of the `filename` and `module_globals`